delta-lake/delta-33x/src/main/scala/com/nvidia/spark/rapids/delta/delta33x/Delta33xProvider.scala

```diff
@@ -72,10 +73,22 @@ object Delta33xProvider extends DeltaProviderBase {
     ).map(r => (r.getClassFor.asSubclass(classOf[RunnableCommand]), r)).toMap
   }
 
-  override protected def toGpuParquetFileFormat(fmt: DeltaParquetFileFormat): FileFormat =
+  override protected def toGpuParquetFileFormat(fmt: DeltaParquetFileFormat): FileFormat = {
+    val enableDVConfig = DeltaConfigs.ENABLE_DELETION_VECTORS_CREATION
+    val isDVEnabled = fmt.metadata.configuration.getOrElse(
+      enableDVConfig.key, enableDVConfig.defaultValue).toBoolean
+    val optimizationsEnabled = if (isDVEnabled) {
+      logWarning(s"Input Delta table has deletion vectors enabled. " +
+        s"Optimizations such as file splitting and predicate pushdown are currently not " +
+        s"supported for this table")
```

Comment:
We are adding yet another warning cluttering the console  printed for all queries and yet it is not actionable for the user.

Reply:
This is bad user experience. We know we need to fix it.